[filter-effects] Property parsing tests

ewilligers opened the pull request:
Test parsing and serialization of the following properties:
- color-interpolation-filters
- filter
- lighting-color

In particular, we test that hue-rotate(0) is accepted.
https://github.com/w3c/fxtf-drafts/issues/228


<!-- Reviewable:start -->

<!-- Reviewable:end -->

ericwilligers committed:
[filter-effects] Property parsing tests

Test parsing and serialization of the following properties:
- color-interpolation-filters
- filter
- lighting-color

In particular, we test that hue-rotate(0) is accepted.
https://github.com/w3c/fxtf-drafts/issues/228

Some filter functions remain to be added to the tests.

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19663)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/289760741)

ericwilligers committed:
[filter-effects] More filter parsing tests

Add tests for drop-shadow() with color before lengths.
https://github.com/w3c/fxtf-drafts/issues/231

Add tests for filter functions with no arguments supplied.

ericwilligers committed:
Tested with Edge and Safari
